mozdownload/scraper.py

```diff
@@ -800,6 +800,8 @@ def get_build_info(self):
         if not self.locale_build:
             self.builds, self.build_index = self.get_build_info_for_index(
                 self.build_index)
+            # Assigned to provide unique timestamp in filename for each file
```

Comment:
I would rephrase this comment to something like: `Always force a timestamp prefix in the filename`. With that fixed we can get this landed. It looks all fine. Thanks.